### ISSUE TITLE
EntityFrameworkCore.Jet 3.1: Unnable to scaffold foreign keys with composed fields

### ISSUE THREAD

**Issue by scosta-kar** [bubibubi/EntityFrameworkCore.Jet#103]
Let me start by thanking you guys for the excellent work on this project.

I am migrating a project that scaffolds an Access Database using EntityFrameworkCore.Jet 2.2.0  to the new 3.1 version.

When trying to scaffold the database, I get the following warning:
```
Could not scaffold the foreign key 'op_3_sections(op_ph_num,op_seq_id)'. A key for 'op_ph_num,op_seq_id' was not found in the principal entity type 'Op2Phases'.
```
The result is we don't get the foreign key mapped to a collection in code.
This seems to be introduced by the upgrade to 3.1. I'm attaching a sample database that replicates this issue.
[sample.zip](https://github.com/bubibubi/EntityFrameworkCore.Jet/files/6586392/sample.zip)

**Comment by lauxjpn** [comment]
@scosta-kar Thanks for reporting this issue!

While we are getting the columns of the FK -> PK mapping in the correct order from the database schema, we sort them later and thereby loosing their original order, which leads to EF Core not matching the columns of the relationship to the primary key of the principal table.

So this is a bug.
The only workaround is to name the columns involved in a way, that they match the order they appear in the PK when sorted by name, which is not really a practical solution.

We will release a fix for this issue shortly.

**Comment by lauxjpn** [comment]
This has been fixed with #104.
Until the release of `3.1.1`, just use the latest [daily build](https://github.com/bubibubi/EntityFrameworkCore.Jet#daily-builds).
